cmd/build-iso.go

```diff
@@ -28,13 +25,7 @@ func NewBuildISOCmd() *cobra.Command {
 			return CheckRoot()
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			path, err := exec.LookPath("mount")
-			if err != nil {
-				return err
-			}
-			mounter := mount.New(path)
-
-			cfg, err := config.ReadConfigBuild(viper.GetString("config-dir"), cmd.Flags(), mounter)
+			cfg, err := config.ReadConfigBuild(viper.GetString("config-dir"), cmd.Flags())
```

Comment:
The title of the PR says it's removed from build-uki. I see its removed from this one too?

Reply:
yeah, everywhere, I started with build-uki but noticed is not needed anywhere, same with chroot